### PR TITLE
SNC: 25 April

### PR DESCRIPTION
**Pull request by Northmoc**
May be a better way to do the Hearse?

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/unlicensed_hearse.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Name:Unlicensed Hearse
+ManaCost:2
+Types:Artifact Vehicle
+PT:*/*
+A:AB$ ChangeZone | Cost$ T | TargetMin$ 0 | TargetMax$ 2 | TargetsFromSingleZone$ True | Origin$ Graveyard | Destination$ Exile | SubAbility$ DBStore | TgtPrompt$ Select up to two target cards in a graveyard | ValidTgts$ Card | RememberChanged$ True | SpellDescription$ Exile up to two target cards from a single graveyard.
+SVar:DBStore:DB$ StoreSVar | SVar$ ExiledAmount | Type$ CountSVar | Expression$ ExiledAmount/Plus.X | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+S:Mode$ Continuous | Affected$ Card.Self | SetPower$ ExiledAmount | SetToughness$ ExiledAmount | Description$ CARDNAME's power and toughness are each equal to the number of cards exiled with it.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/unlicensed_hearse.txt:8]: "exiled with" means cards in exile itself exiled it
that also means that if the exiled card leaves exile, it isn't exiled with anymore

(that later needs to be connected to linked ability anyway)

File: forge-gui/res/cardsfolder/upcoming/rocco_cabaretti_caterer.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Rocco, Cabaretti Caterer
+ManaCost:X R G W
+Types:Legendary Creature Elf Druid
+PT:3/1
+T:Mode$ ChangesZone | ValidCard$ Card.Self+wasCast | Destination$ Battlefield | Execute$ TrigChangeZone | OptionalDecider$ You | TriggerDescription$ When CARDNAME enters the battlefield, if you cast it, you may search your library for a creature card with mana value X or less, put it onto the battlefield, then shuffle.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/rocco_cabaretti_caterer.txt:5]: wasCastByYou

File: forge-gui/res/cardsfolder/upcoming/unlicensed_hearse.txt (hunk 1 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/unlicensed_hearse.txt]: It will be great to streamline all these ExiledWithSource+Remembered cards

**Review by Hanmac — APPROVED** [review]
i will refactor the ExiledWith later

**Pull request merged**